src/app/components/MainContent/index.jsx

```diff
@@ -41,7 +41,13 @@ const MainContent = ({ data }) => {
 MainContent.propTypes = {
   data: shape({
     model: shape({
-      blocks: arrayOf(any),
+      blocks: shape({
+        headline: PropTypes.object.isRequired,
+        subheading: PropTypes.object.isRequired,
+        text: PropTypes.object.isRequired,
```

Comment:
These need to be defined more than just an `object` - they should have a shape.

We are defining propTypes for child components [here](https://github.com/bbc/simorgh/blob/latest/src/app/helpers/proptypes.js#L13-L25), and you use those.

This is the same for `headline`, and it uses `textPropTypes`.